src/drafts/index.ts

```diff
@@ -1,9 +1,12 @@
-/** This is the place where we keep components that are not part of the public
- *  api yet (not in main bundle). We don't recommend using it in production.
+/**
+ * This is the place where we keep components that are not part of the public
+ * api yet (not in main bundle). We don't recommend using it in production.
  *
- *  But, they are published on npm and you can import them for experimentation/feedback.
- *  example: import {ActionList} from '@primer/react/drafts
+ * But, they are published on npm and you can import them for experimentation/feedback.
+ * example: import {ActionList} from '@primer/react/drafts
  */
+
+export {DataTable, Table} from '../DataTable'
```

Comment:
Should we export any types?

Reply:
Makes sense to me as part of the definition of done. To implement this, should I include props as exports for any exported component?